src/main/views/webContentEvents.ts

```diff
@@ -169,6 +170,11 @@ export class WebContentsEventManager {
                 return {action: 'allow'};
             }
 
+            // Allow plugins to open blank popup windows.
+            if (parsedURL.toString() === 'about:blank') {
```

Comment:
We are missing the check for plugin URLs:
```suggestion
            if (isPluginUrl(serverURL, parsedURL) && parsedURL.toString() === 'about:blank') {
```

This should also probably be done further down.

Reply:
@devinbinnie I think the point is to allow creating popouts from the webapp, not necessarily a plugin path. If we add this check then it won't really work the way I am thinking.

Reply:
Is there a reason we can't forward them to a link in the `plugins` route that would open this?
The reason I'd rather have that is that I wouldn't want any part of the app opening these windows, for example, if someone was able to post a message that was able to execute some JS, they could force open a malicious popup.

Reply:
> Is there a reason we can't forward them to a link in the plugins route that would open this?

Mostly it's the requirement to open basic `about:blank` popups and not loading any server side content under different paths. 

> The reason I'd rather have that is that I wouldn't want any part of the app opening these windows, for example, if someone was able to post a message that was able to execute some JS, they could force open a malicious popup.

This can happen today in a browser context, the plugin can open whatever. But in Desktop at least we'd be ensuring they could only open `about:blank` which I can't see being any more malicious than the plugin (and webapp as a consequence) already is.